docs/quickstart.md

```diff
@@ -0,0 +1,154 @@
+---
+post_title: Spark Quickstart
+menu_order: 10
+feature_maturity: stable
+enterprise: 'no'
+---
+
+This tutorial will get you up and running in minutes with Spark. You will install the DC/OS Spark.
```

Comment:
@mgummelt should this be changed globally (i.e. no instances of simply `DC/OS Spark`)?